tests/X86/Run.cpp

```diff
@@ -487,6 +487,21 @@ static void ResetFlags(void) {
   asm("push %0; popfq;" : : "m"(gRflagsInitial));
 }
 
+// clear the exception flags in mxcsr
+// *and* set MXCSR to ignore denormal exceptions
+// this is done by newer (after 2015) glibcs
```

Comment:
Maybe worth specifically mentioning the routine with the error: "// this is done properly by std::fsetenv in glibc after ~2015, but was broken in earlier versions."